dlc_developer_config.toml

```diff
@@ -13,22 +13,22 @@ benchmark_mode = false
 
 [build]
 # Frameworks for which you want to disable both builds and tests
-skip_frameworks = []
+skip_frameworks = ["mxnet", "tensorflow", "huggingface_tensorflow", "huggingface_pytorch", "autogluon"]
 # Set to false in order to remove datetime tag on PR builds
-datetime_tag = true
+datetime_tag = false
 # Note: Need to build the images at least once with datetime_tag = false
 # before disabling new builds, or tests will fail
 do_build = true
 
 [test]
 # On by default
 sanity_tests = true
-ecs_tests = true
-eks_tests = true
-ec2_tests = true
+ecs_tests = false
+eks_tests = false
+ec2_tests = false
 
 # Off by default
-sagemaker_tests = false
-efa_tests = false
+sagemaker_tests = true
```

Comment:
@roywei - if you consume the latest master, and set `sagemaker_remote_tests = "rc"` - you can also test RC tests

Reply:
thanks! do you mean adding another line like this?
```
sagemaker_tests = true
sagemaker_remote_tests = rc
```

Reply:
no, let me help resolve the conflicts and I'll show you